### ISSUE TITLE
Question about the data preprocessing step

### ISSUE THREAD

**Issue by TuanBC** [Jiaxin-Ye/TIM-Net_SER#7]
Thanks for the great research. Could you also share the source code of the data-preprocessing and train-dev-test split code? I want to make sure if the split strategy in the paper is fair and no contamination of training set happened during the testing procedure.

**Comment by hwang9u** [comment]
I'm curious too!

**Comment by Jiaxin-Ye** [comment]
Thanks for your attention! But I am so sorry. I don't seem understand your question. For data pre-processing, we first extract MFCC for all speech signals in the corpus without distinguishing any labels or other information following the same process, and then store them all in .npy format. For training progress, we take a random 10-fold cross-validation approach to split the training and test sets, and the test set of each fold does not have data overlap with the training set. Specifically, you can see the source code (**_Model.py_**) as follows:
https://github.com/Jiaxin-Ye/TIM-Net_SER/blob/73488c8db586e749c46a63678e858b05d6d2f1a1/Code/Model.py#L99
https://github.com/Jiaxin-Ye/TIM-Net_SER/blob/73488c8db586e749c46a63678e858b05d6d2f1a1/Code/Model.py#L102
